pkg/crc/preflight/preflight_checks_darwin.go

```diff
@@ -182,3 +185,27 @@ func addFileWritePermissionToUser(filename string) error {
 
 	return nil
 }
+
+func removeCRCHyperkitMachine() error {
+	err := os.RemoveAll(filepath.Join(constants.MachineInstanceDir, "crc"))
+	if err != nil {
+		logging.Debugf("Failed to remove hyperkit machine directory: %v", err)
+		return err
+	}
+	return nil
+}
+
+func stopCRCHyperkitProcess() error {
+	path, err := exec.LookPath("pkill")
+	if err != nil {
+		logging.Debugf("Could not find 'pkill'. %v", err)
+		return fmt.Errorf("Could not find 'pkill'. %v", err)
+	}
+	cmd := exec.Command(path, "-f", filepath.Join(constants.CrcBinDir, "hyperkit"))
+	err = cmd.Run()
+	if err != nil {
```

Comment:
If you like it: 
`if err := exec.Command(path, "-f", filepath.Join(constants.CrcBinDir, "hyperkit")).Run(); err != nil`

Maybe use crcos.RunWithDefaultLocale ?

Reply:
this should be ok without `crcos.RunWithDefaultLocale`, as we are not using the o/p from this command..

Reply:
In my  opinion, we should never use `exec.Command`/`cmd.Run`, and always go through `RunWithDefaultLocale`. Even if it does not matter here, this could be copied and pasted to do something different where it would matter, which is why I prefer to use `RunWithDefaultLocale` most of the time (and also consistency, `exec.Command.Run()` and `RunWithDefaultLocale` looks very different)